### PR TITLE
[New Touch Control Layout] Cyberpunk 2077

### PR DESCRIPTION
**Pull request by evanpaul14**
Screenshot 2024-03-26 at 7 56 17 PM

### CODE REVIEW

File: touch-layouts/layouts/cyberpunk-2077.json [hunk 1]
```diff
@@ -0,0 +1,371 @@
+{
+  "schema_version": 1,
+  "layouts": {
+    "LAYOUT_ID": {
```
Comment (redphx) [inline touch-layouts/layouts/cyberpunk-2077.json]: This should be changed to `custom` (same value as `default_layout` in other file)

File: touch-layouts/layouts/cyberpunk-2077.json [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+          "inner": [
+            {
+              "type": "joystick",
+              "axis": {
+                "input": "axisXY",
+                "output": "leftJoystick"
+              },
+              "action": "leftThumb",
+              "relative": false,
+              "actionThreshold": 1,
```
Comment (redphx) [inline touch-layouts/layouts/cyberpunk-2077.json]: This number should be higher, at least 2

**Review by redphx — CHANGES_REQUESTED** [review]
You don't need to close old pull request, just edit your repo.

File: touch-layouts/layouts/cyberpunk-2077.json (hunk 2 above)
Comment (evanpaul14) [reply touch-layouts/layouts/cyberpunk-2077.json]: Why is that? It worked fine in my testing.

Comment (redphx) [reply touch-layouts/layouts/cyberpunk-2077.json]: So you don't activate it by accident

**Pull request merged**